test-project/Packages/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "com.unity.ext.nunit": "1.0.0",
-    "com.unity.ide.rider": "1.0.8",
+    "com.unity.ide.rider": "1.1.0",
```

Comment:
I thought we were going to remove this?

Reply:
we were - but then I ran into other issues with Rider when this wasn't there at all